kafka-connect-debezium/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM strimzi/kafka:latest-kafka-2.3.0
```

Comment:
for now we are using Strimzi 0.12.1 (which is the base for AMQ Streams 1.2.0) so Kafka cannot be 2.3.0